test/composite_parser.js

```diff
@@ -137,17 +142,18 @@ describe('Composite parser', function() {
           type: rowParser,
         });
 
-      var buffer = Buffer.alloc(1 + 10 * (1 + 5 * 4));
+      var buffer = new Buffer(1 + 10 * (1 + 5 * 4));
```

Comment:
Please use `Buffer.alloc` instead of the constructor.

> (node:55) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.